network-api/networkapi/buyersguide/models.py

```diff
@@ -134,6 +136,13 @@ class Product(ClusterableModel):
         help_text='Review date of this product',
     )
 
+    @property
+    def is_current(self):
+        d = self.review_date
+        review = datetime(d.year, d.month, d.day)
+        cutoff = datetime(2019, 6, 1)
```

Comment:
Is June 1st or July 1st our cutoff date? I think month for Python's `datetime` ranges from 1-12 instead of 0-11?

Reply:
I intended June here